### DISCUSSION TITLE
GitHub Actions CI: Enable Testing for ARM64 platform

### DISCUSSION THREAD

**Discussion by odidev** [minio/minio#14708] [Q&A] [answered]
Hi Team, 
I have been working on testing minio features on arm64 platform and they are working successfully. Are you planning on running functional tests for arm64 platform as well on GitHub Actions CI?

**Comment by klauspost** [comment, marked as answer]
@odidev I think we would all like that, but github doesn't offer arm64 platforms for actions. QEMU will be way too slow to be useful.

**Comment by odidev** [comment]
Hi @klauspost,
Would you be interested in adding ARM64 tests in GitHub Actions using self-hosted arm64 runners. If interested, We can get access to ARM H/W through [WorksOnArm](https://github.com/WorksOnArm).

**Reply by odidev** [reply]
Gentle reminder.

**Reply by odidev** [reply]
Hi @klauspost,
Any updates on the above?
